packages/server/src/subscriptions/websockets.test.ts

```diff
@@ -37,7 +36,8 @@ describe('WebSockets Subscriptions', () => {
     app = express();
     config = await loadTestConfig();
     config.heartbeatEnabled = false;
-    server = await initApp(app, config);
+    server = await withTestContext(() => initApp(app, config));
```

Comment:
Hm, do you know what in `initApp` required `withTestContext`?

I think we went overboard with `withTestContext`

In this PR, I cleaned up a bunch of cases that required `withTestContext`: https://github.com/medplum/medplum/pull/4019

If you think about it, `initApp` should not require a request context.

Reply:
That's correct, I think I had added this when throwing everything at the wall to get tests to pass but forgot to remove this one 👍